docs/tutorials/testfiles/crp-availability.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: placement.kubernetes-fleet.io/v1
+kind: ClusterResourcePlacement
+metadata:
+  name: crp-availability
+spec:
+  policy:
+    placementType: PickN
+    numberOfClusters: 2
+    affinity:
+      clusterAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          clusterSelectorTerms:
+            - propertySelector:

```

Comment:
I think this should be the original crp where this crp selects the labels instead.
```
 - labelSelector:
      matchLabels:
        fleet.azure.com/location: westus
```

Reply:
I've updated this section to match the original code snippet